package.json

```diff
@@ -59,7 +59,8 @@
     "test": "npm run lint && npm run test:server && npm run test:client",
     "test:client": "karma start --single-run",
     "test:server": "mocha --timeout 20000 --compilers js:babel-core/register test/bootstrap.test.js test/controller/**/*.js",
-    "test:watch": "concurrently \"mocha -w --compilers js:babel-core/register\" \"karma start\""
+    "test:watch": "concurrently \"mocha -w --compilers js:babel-core/register\" \"karma start\"",
+    "version": "node -e 'console.log(require(\"./package\").version)'"
```

Comment:
I'm having trouble verbalising but... I don't like this. I don't know why, but I don't see how you can prefer this to the other solution. As @snowpheonix says, we wouldn't be running it manually anyway (and definitely shouldn't) and this is a horrible solution to the problem. We are using a console log as an official part of our production build? This seems so fragile. So many bits have to work together for this and I really don't like it.